install/scripts/aws-up.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+set -u
+
+# Required env variables:
+: "$AWS_ACCOUNT_ID $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY"
+install_operator="${INSTALL_OPERATOR:-yes}"
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+kubernetes_dir=${script_dir}/../kubernetes
+
+EKSCTL_ENABLE_CREDENTIAL_CACHE=1
+export CLUSTER_NAME=substratus
+export REGION=us-west-2
+export ARTIFACTS_REPO_NAME=${CLUSTER_NAME}
+export ARTIFACTS_BUCKET_NAME=${AWS_ACCOUNT_ID}-${CLUSTER_NAME}-artifacts
+tempout=$(mktemp)
+
+aws s3 mb s3://${ARTIFACTS_BUCKET_NAME} \
+  --region ${REGION} >/dev/null || true
+
+aws ecr create-repository \
+  --repository-name ${ARTIFACTS_REPO_NAME} \
+  --region ${REGION} >/dev/null || true
+
+envsubst <${kubernetes_dir}/aws/eks-cluster.yaml.tpl >${kubernetes_dir}/aws/eks-cluster.yaml
+eksctl create cluster -f ${kubernetes_dir}/aws/eks-cluster.yaml ||
+  eksctl upgrade cluster -f ${kubernetes_dir}/aws/eks-cluster.yaml
+
+aws iam create-service-linked-role \
+  --aws-service-name spot.amazonaws.com || true
+
+aws eks update-kubeconfig \
```

Comment:
I think you get this for free from `eksctl`?

Reply:
Yes but I wonder if it becomes your context. Docs say:

> After the cluster has been created, the appropriate kubernetes configuration will be added to your kubeconfig file.